libiocage/lib/helpers.py

```diff
@@ -102,6 +102,26 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+# This little helper function will try to parse booleans from strings
+# On success, it returns the parsed boolean, on failure it returns the
+# `default`. By default, `default` is `False`.
+def parse_bool(data, default=False):
+    if isinstance(data, bool):
+        return data
+    if isinstance(data, str):
+        if data.lower() in ["yes", "true", "on", "1"]:
```

Comment:
If `int` is valid, why are we checking for `str` only? It should be a separate `elif` imo in that case.